### PR TITLE
Unstable

### PR DESCRIPTION
**Pull request by huminzhi**
@bhariharan @kchitalia @wmathurin @helenren FYI.

### CODE REVIEW

**Comment by kchitalia** [comment]
Is this related to the CordovaPlugin issue?

**Comment by huminzhi** [comment]
@kchitalia sorry not yet..

**Comment by kchitalia** [comment]
Ok, I got confused, sorry :). Looks like this will resolve https://github.com/forcedotcom/SalesforceMobileSDK-iOS/issues/1510

👍

**Comment by bhariharan** [comment]
👍

File: libs/SalesforceRestAPI/SalesforceRestAPI/Classes/SFRestAPI+Files.m [hunk 1]
```diff
@@ -40,41 +40,41 @@
 @implementation SFRestAPI (Files)
 
 - (SFRestRequest *) requestForOwnedFilesList:(NSString *)userId page:(NSUInteger)page {
-    NSString *path = [NSString stringWithFormat:@"/%@/chatter/users/%@/files", self.apiVersion, (userId == nil ? ME : userId)];
```
Comment (wmathurin) [inline libs/SalesforceRestAPI/SalesforceRestAPI/Classes/SFRestAPI+Files.m]: Should we do (api >36 ? @"/connect/files/..." : @"/chatter/users/...")?

Comment (bhariharan) [inline libs/SalesforceRestAPI/SalesforceRestAPI/Classes/SFRestAPI+Files.m]: Default is ```v36.0```, so we should be okay. There's no reason for users to downgrade API version on new SDK. Typically internal consumers upgrade to pre-rel.

Comment (wmathurin) [inline libs/SalesforceRestAPI/SalesforceRestAPI/Classes/SFRestAPI+Files.m]: That way the URL would be correct in all cases

Comment (bhariharan) [inline libs/SalesforceRestAPI/SalesforceRestAPI/Classes/SFRestAPI+Files.m]: I'll fix it.

**Pull request merged**